OpenSim/Simulation/Model/Model.cpp

```diff
@@ -2220,8 +2220,31 @@ SimTK::Vec3 Model::calcMassCenterVelocity(const SimTK::State &s) const
 SimTK::Vec3 Model::calcMassCenterAcceleration(const SimTK::State &s) const
 {
     getMultibodySystem().realize(s, Stage::Acceleration);
-    return getMatterSubsystem().calcSystemMassCenterAccelerationInGround(s);    
+    return getMatterSubsystem().calcSystemMassCenterAccelerationInGround(s);
 }
+/**
```

Comment:
These Doxygen-style comments don't get picked up from the .cpp file (only the headers). See, for example, [the (blank) documentation for `Model::calcMassCenterPosition()`](https://simtk.org/api_docs/opensim/api_docs/classOpenSim_1_1Model.html#a937ccd739df45c80c48a308a90aaf4cd) (notably, the potentially useful comment "Return the position vector of the system mass center, measured from the Ground origin, and expressed in Ground." does not appear). Perhaps at least put the Doxygen comments for your new methods in the header file, and create an Issue to move the others.

Reply:
Good catch! I can move all these over.